index.html

```diff
@@ -0,0 +1,213 @@
+                </ul>
+              </div>
+              <p class="text-card" >
+                Exploring the future of media in Facebook's first Virtual Reality<br>app; a place to discover and enjoy 360 photos and videos on<br>Gear VR..
+              </p>
+              <div class="skills-card-container">
+                <ul class="skills-card">
+                  <li class="skills">html</li>
+                  <li class="skills">Ruby on rails</li>
+                  <li class="skills">css</li>
+                  <li class="skills">javaScript</li>
+                </ul>
+              </div>
+              <button class="button-see-alter" type="button">See Projects</button>
+            </div>
+          </section>
+          <section class="card">
+            <div class="block-image"><img src="image/card-4.png" alt="Missed Image" class="image-card"></div>
+            <div class="block-card reverse">
+              <h2 class="header-card">Uber Navigation</h2>
+              <div class="title-card-container">
+                <ul class="title-card">
+                  <li class="exclude-list">Uber</li>
+                  <li>Lead Developer</li>
+                  <li>2022</li>
+                </ul>
+              </div>
+              <p class="text-card" >
+                A smart assistant to make driving more safe, efficient, and fun<br>by unlocking your most expensive computer: your car.
+              </p>
+              <div class="skills-card-container">
+                <ul class="skills-card">
+                  <li class="skills">html</li>
+                  <li class="skills">Ruby on rails</li>
+                  <li class="skills">css</li>
+                  <li class="skills">javaScript</li>
+                </ul>
+            </div>
+              <button class="button-see" type="button">See Projects</button>
+            </div>
+          </section>
+        </article>
+    <!-- About Start -->
+        <article id="about-content">
+          <section class="content-about-container">
+            <div class="block-left">
+              <h2 class="title-about">About<br>Myself</h2>
+              <p class="text-about"> Hello I’m a software developer! I can help you build a<br>product , feature or website Look through some of my<br>work and experience! If you like what you see and have<br>a project you need coded, don’t hestiate to contact me.</p>
+              <h4 class="text-social">LETS'S CONNECT</h4>
+              <ul class="icons-foot contacts-social">
+                <li><a href="#"><i class="fa fa-github" aria-hidden="true"></i></a></li>
+                <li><a href="#"><i class="fa fa-linkedin" aria-hidden="true"></i></a></li>
+                <li><a href="#"><i class="fa fa-angellist" aria-hidden="true"></i></a></li>
+                <li><a href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a></li>
+                <li><a href="#"><i class="fa fa-medium" aria-hidden="true"></i></a></lskill-otheri>
```

Comment:
- [ ] I think it will be better if you align your about me description properly to the left to match the Figma Design you chose 👍🏽 

| Figma Design | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/193326940-eab1a1bd-2b47-4419-96ef-b6796fc63109.png)  | ![image](https://user-images.githubusercontent.com/63473584/193327026-aee2c078-0c8c-49f6-a774-1d27b73b7d2c.png)  |